components/Buttons/LikeButton.vue

```diff
@@ -1,39 +1,33 @@
 <template>
-  <v-btn :dark="dark" icon @click.stop.prevent="toggleFavorite">
-    <v-icon :class="fav ? 'red--text' : ''">
-      {{ fav ? 'mdi-heart' : 'mdi-heart-outline' }}
+  <v-btn icon @click.stop.prevent="toggleFavorite">
```

Comment:
Why dark was removed? This makes the button invisible in cards

Reply:
I'll have another look, but as far as I could see we weren't using it anywhere

Reply:
Yeah checked again. The dark prop for like-button isn't used anywhere.

https://github.com/jellyfin/jellyfin-vue/blob/master/components/Buttons/QueueButton.vue#L45

https://github.com/jellyfin/jellyfin-vue/blob/master/components/Item/Card/Card.vue#L81

https://github.com/jellyfin/jellyfin-vue/blob/master/components/Layout/AudioControls.vue#L129

https://github.com/jellyfin/jellyfin-vue/blob/master/components/Players/DraggableQueue.vue#L42

https://github.com/jellyfin/jellyfin-vue/blob/master/components/Players/TrackList.vue#L43

https://github.com/jellyfin/jellyfin-vue/blob/master/pages/item/_itemId/index.vue#L58

Reply:
@Camc314 This is a misstep from my rebase of #609. Can you add it to card, please?

If you go to light mode in the current state, the like button is not visible, so we need it for cards. No problems for the rest of the UI.

Reply:
I've added the prop back to the card